ui/src/logs/actions/index.ts

```diff
@@ -67,27 +162,66 @@ export const setNamespaces = (
   },
 })
 
-export const setTimeRange = (timeRange: TimeRange): SetTimeRangeAction => ({
-  type: ActionTypes.SetTimeRange,
-  payload: {
-    timeRange,
-  },
-})
+export const setTimeRange = (timeRange: TimeRange) => async dispatch => {
+  dispatch({
+    type: ActionTypes.SetTimeRange,
+    payload: {
+      timeRange,
+    },
+  })
+  dispatch(setHistogramQueryConfig())
+}
+
+export const populateNamespaces = (proxyLink: string) => async dispatch => {
+  const namespaces = await getDatabasesWithRetentionPolicies(proxyLink)
```

Comment:
could you add error handling?

Reply:
we haven't flushed out the page far enough to have error messaging, will be coming in a future PR.

Reply:
👍